configure.ac

```diff
@@ -48,6 +48,13 @@ AC_COMPILE_IFELSE(
 	 AC_MSG_RESULT([yes])],
 	[AC_MSG_RESULT([no])])
 
+AC_MSG_CHECKING([whether aio_init is supported])
+AC_COMPILE_IFELSE(
+        [AC_LANG_PROGRAM([[#include <aio.h>]], [[aio_init(NULL);]])],
+        [AC_MSG_RESULT([no])],
+        [AC_DEFINE([HAVE_AIO_INIT], [1], [Define if aio_init is available]),
```

Comment:
I'm confused now. If I invert the "yes" and the "no" statements, then it fails on both ArchLinux (glibc) and Alpine (musl). But reading the [documentation](https://www.gnu.org/software/autoconf/manual/autoconf-2.61/html_node/Generating-Sources.html), it should create a file that includes `<aio.h>` and try to run `aio_init(NULL);`. So it should go the other way round, right? :confused: